java/adapter/jdbc/pom.xml

```diff
@@ -17,6 +17,7 @@
         <groupId>org.apache.arrow</groupId>
         <artifactId>arrow-java-root</artifactId>
         <version>0.10.0-SNAPSHOT</version>
+        <relativePath>../..</relativePath>
```

Comment:
I'm more used to specifying this as `../../pom.xml`, would you mind changing to this?

Reply:
Fixed it.